tasks/sanity-check.yml

```diff
@@ -11,6 +11,7 @@
     - "{{ jenkins_jcasc_config_file }}"
```

Comment:
`jenkins_jcasc_config_file` is allowed to be empty, so we should remove this line.

Reply:
Yeah, I think I meant to remove this line but removed the plugins file by accident. 😅